webdav3/client.py

```diff
@@ -235,18 +235,22 @@ def valid(self):
         return True if self.webdav.valid() else False
 
     @wrap_connection_error
-    def list(self, remote_path=root):
+    def list(self, remote_path=root, getinfo = False):
         """Returns list of nested files and directories for remote WebDAV directory by path.
         More information you can find by link http://webdav.org/specs/rfc4918.html#METHOD_PROPFIND
 
         :param remote_path: path to remote directory.
-        :return: list of nested file or directory names.
+        :param getinfo: path and element info to remote directory, like cmd 'ls -l'.
+        :return: list of nested file or directory names and info.
```

Comment:
@huangganggui please rewrite more the return description for more clarity how `get_info` impacts that and structure of dictionary return. 
Ex: "if get_info=False it returns list of ... otherwise list of infos with structure ... "